common.fabric.mk

```diff
@@ -361,5 +361,9 @@ compile-phylum-%: $$(shell find -L ../$$* -name "*.lisp" 2>/dev/null | grep -Fv
 	cp $^    ./build/phylum_$*/src/
 	cd       ./build/phylum_$*/src && ls && rm -f ./../phylum.zip && zip ./../phylum.zip $(notdir $^)
 
-${CC_PATH}:
+${CC_PATH}: ${PRESIGNED_PATH}
 	${PROJECT_REL_DIR}/scripts/obtain-cc.sh
+	touch $@
```

Comment:
Do you really want to `touch $@` here? CC_PATH isn't a "dummy" path is it? It's the actual path of the cc tarball. right?

Reply:
So apparently the mod time from the file I was downloading from s3 was interfering with `make`, and this fixed it. Have you seen that before?

Reply:
Oh. Huh. I have never noticed that.. But I guess the s3 cli might be trying to set the mod time to be helpful?